### PR TITLE
Anoles Genome

### PR DESCRIPTION
**Pull request by Vcaudill**
will need to update recombination rate
Added mutation rate, population size, and generation time

### CODE REVIEW

File: stdpopsim/catalog/AnoCar/species.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    reasons={
+        stdpopsim.CiteReason.POP_SIZE,
+        stdpopsim.CiteReason.MUT_RATE,
+        stdpopsim.CiteReason.REC_RATE,
+    },
+)
+
+# No recombination rate yet for this species.
+# I might use a bird like a chicken
+
```
Comment (jeromekelleher) [inline stdpopsim/catalog/AnoCar/species.py:25]: Do you have an estimate that we could use @Vcaudill?

**Review by jeromekelleher — COMMENTED** [review]
Looks good, thanks @Vcaudill. We will need some guess at the recombination rate I think.

File: stdpopsim/catalog/AnoCar/species.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    reasons={
+        stdpopsim.CiteReason.POP_SIZE,
+        stdpopsim.CiteReason.MUT_RATE,
+        stdpopsim.CiteReason.REC_RATE,
+    },
+)
+
+# No recombination rate yet for this species.
+# Author of BourgeoisEtAl is sending the recombination map
+# Recombintation rate of 1e-8
```
Comment (petrelharp) [inline stdpopsim/catalog/AnoCar/species.py:25]: don't need this line - it's redundant with the next one

File: stdpopsim/catalog/AnoCar/species.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    "5": 1e-8,
+    "6": 1e-8,
+    "LGa": 1e-8,
+    "LGb": 1e-8,
+    "LGc": 1e-8,
+    "LGd": 1e-8,
+    "LGf": 1e-8,
+    "LGg": 1e-8,
+    "LGh": 1e-8,
+    "MT": 1e-8,
```
Comment (petrelharp) [inline stdpopsim/catalog/AnoCar/species.py:42]: this should be 0

File: stdpopsim/catalog/AnoCar/species.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        stdpopsim.CiteReason.MUT_RATE,
+        stdpopsim.CiteReason.REC_RATE,
+    },
+)
+
+# No recombination rate yet for this species.
+# Author of BourgeoisEtAl is sending the recombination map
+# Recombintation rate of 1e-8
+# Placeholder rate of 1cM/Mb used
+
```
Comment (petrelharp) [inline stdpopsim/catalog/AnoCar/species.py:27]: maybe do an `_overall_rate` thing like with mutation rate?

File: stdpopsim/catalog/AnoCar/species.py [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    ensembl_id="anolis_carolinensis",
+    name="Anolis carolinensis",
+    common_name="Anole lizard",
+    genome=_genome,
+    generation_time=2.5,
+    # they live between 1-2 years after they are able to mate
+    # they mature 8 to 9 months after they are born
+    # can live up to 8 years in captivity
+    population_size=5.34e6,
+    # estimated by using different recombination rates found in BourgeoisEtAl
```
Comment (petrelharp) [inline stdpopsim/catalog/AnoCar/species.py:83]: this line is not relevant any more, right?

**Comment by petrelharp** [comment]
LGTM!

**Review by petrelharp — APPROVED** [review]
LGTM, thanks!

**Pull request merged**